features/staging_commands.feature

```diff
@@ -30,53 +30,6 @@ Feature: Enable command behaviour when attached to an Ubuntu Pro staging subscri
         \s*\*\*\* .* 500
         \s*500 https://esm.staging.ubuntu.com/apps/ubuntu <release>-apps-security/main amd64 Packages
         """
-        When I run `mkdir -p /var/lib/ubuntu-advantage/messages` with sudo
```

Comment:
I think I am missing something, but why are we removing that part of the test ?

Reply:
The new tests test the full process of creating the messages and the apt hook filling in these variables. So I didn't think we needed these anymore, which only test how the apt hook fills in variables and concatenates files.

But I can add them back if we think they add value!

Reply:
Fair enough, just wanted to double check. No problem removing that